.github/workflows/actions-tagger.yaml

```diff
@@ -0,0 +1,13 @@
+name: actions-tagger
```

Comment:
By the way, should I rename `.yaml` to `.yml` for consistency?

Why I used `.yaml` here is the official YAML recommends so.
http://yaml.org/faq.html